src/stores/team/useTeamUsers.tsx

```diff
@@ -0,0 +1,19 @@
+import useSWR, { SWRResponse } from 'swr';
+import { User } from '~/domains';
+import { createMockUser } from '~/mock';
+
+/**
+ * チームに所属しているUserを取得するSWR
+ * @returns data 所属しているUser
+ * @returns isValidating 取得中を表す boolean
+ * @returns error エラー
+ * @returns mutate データの更新関数
+ */
+export const useTeamUsers = ({ teamId }: { teamId?: string }): SWRResponse<User[], Error> => {
+  const key = teamId ? `/teams?teamId=${teamId}` : null;
```

Comment:
あ、これ改めて考えると team を teamId をもとに取得しているエンドポイントですね

正しくは
```suggestion
  const key = teamId ? `/teams/${teamId}/users` : null;
```
でした

Reply:
実装しました

https://github.com/proeco/proeco-back/blob/master/src/routes/teams/getTeamUsers.ts#L16-L21